cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/verification/FlowVerification.java

```diff
@@ -60,7 +61,7 @@
    * @return An instance of {@link VerifyResult} depending of status of verification.
    */
   @Override
-  public VerifyResult verify(Id.Application appId, final FlowSpecification input) {
+  public VerifyResult verify(ApplicationId appId, final FlowSpecification input) {
```

Comment:
how about instead changing ``SimpleQueueSpecificationGenerator``? It has only very few usages, some of which already have ``ApplicationId``?